conf.yaml

```diff
@@ -190,6 +189,12 @@ repository:
                 -
                     action: assign_tag
                     action_value: 'popularity/low'
+                 -
+                    action: remove_tag
+                    action_value: 'popularity/medium
```

Comment:
If it isn't too much to ask, would you mind converting this to be similar to https://github.com/erasche/p4/pull/12/files#diff-4a701a5adb4359c6abf9b8e1cb38819fR110 i.e. a regex?